File: app/src/main/java/com/apps/asritha/fivecards/Shuffle.java
package com.apps.asritha.fivecards;

import java.util.Comparator;

/**
 * Created by Asritha on 21-05-2016.
 */
public class Shuffle implements Comparator<Card> {

    @Override
    public int compare(Card c1, Card c2)
    {
        if(c1.rank().ordinal() > c2.rank().ordinal())
        {
            return 1;
        }

        else
            return -1;
    }
}

File: app/src/main/java/com/apps/asritha/fivecards/Shuffle.kt
package com.apps.asritha.fivecards

class Shuffle : Comparator<Card> {
    override fun compare(c1: Card, c2: Card): Int {
        return if (c1.rank()!!.ordinal > c2.rank()!!.ordinal) 1 else -1
    }
}
